node/subscriptions-with-stripe/README.md

```diff
@@ -0,0 +1,110 @@
+# 💳 Node.js Stripe Subscriptions Function
+
+Integrates Stripe subscriptions into your Appwrite project. Collect card payment with the `/checkout` endpoint and check the status of a user subscription using the `Subscriptions` collection.
+
+## 🧰 Usage
+
+### `POST /subscribe`
+
+This endpoint initiates a Stripe checkout session for a subscription. The user ID is fetched from the headers of the request. If the user ID is not found or a Stripe checkout session could not be created, the request will be redirected to a cancel URL.
+
+**Parameters**
+
+| Name               | Description            | Location | Type   | Sample Value |
+| ------------------ | ---------------------- | -------- | ------ | ------------ |
+| x-appwrite-user-id | User ID from Appwrite. | Header   | String | 642...7cd    |
+| Content-Type       | The content type of the request body | Header   | `application/json` | N/A |
+| successUrl         | The URL to redirect to after a successful payment. | Body   | String | https://example.com/success    |
+| failureUrl         | The URL to redirect to after a cancelled payment attempt. | Body   | String | https://example.com/failure    |
+
+**Response**
+
+Sample `303` Response:
+
+The response is a redirect to the Stripe checkout session URL or to the cancel URL if an error occurs
+
+```text
+Location: https://checkout.stripe.com/pay/cs_test_...#fidkdWxOYHwnP
+```
+
+```text
+Location: https://mywebapp.com/cancel
+```
+
+### `POST /webhook`
+
+This endpoint is a webhook that handles two types of events from Stripe: `customer.subscription.created` and `customer.subscription.deleted`. It validates the incoming request using the Stripe's validateWebhook method. If the validation fails, a `401` response is sent.
+
+**Parameters**
+
+| Name | Description                  | Location | Type   | Sample Value                                                          |
+| ---- | ---------------------------- | -------- | ------ | --------------------------------------------------------------------- |
+| None | Webhook payload from Stripe. | Body     | Object | [See Stripe documentation](https://stripe.com/docs/api/events/object) |
+
+**Response**
+
+Sample `200` Response:
+
+In case of `customer.subscription.created` event, it creates a new subscription for the user.
+In case of `customer.subscription.deleted` event, it deletes the subscription for the user.
```

Comment:
```suggestion
In case of `customer.subscription.created` event, it gives user `subscriber` label.
In case of `customer.subscription.deleted` event, it takes `subscriber` label away from user.
```